src/components/ImportForm/ReviewSection/RuntimeSelector.tsx

```diff
@@ -45,22 +46,43 @@ export const RuntimeSelector: React.FC<RuntimeSelectorProps> = ({ detectedCompon
   const [selectedRuntime, setSelectedRuntime] = React.useState(null);
 
   const DetectingRuntime = 'Detecting runtime...';
+  const dockerFileSample = React.useMemo(
+    () => ({
+      key: 'docker-build',
+      value: 'Dockerfile',
+      icon: <DockerIcon color="#2496ed" />,
+      runtime: { name: 'Dockerfile' },
+    }),
+    [],
+  );
```

Comment:
This can be a const outside of the component instead of a memo.

Reply:
updated.